<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="AMTLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="AMTLineEdit" name="authorLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="AMTEntryFileInput" name="fileInput" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="previewPageLabel">
       <property name="text">
        <string>Preview page:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="previewPageSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AMTLineEdit</class>
   <extends>QLineEdit</extends>
   <header>amt/views/customWidgets/amtinputwidgets</header>
  </customwidget>
  <customwidget>
   <class>AMTEntryFileInput</class>
   <extends>QWidget</extends>
   <header>amt/views/customWidgets/amtinputwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
